config/default-spinnaker-local.yml

```diff
@@ -143,7 +146,10 @@ providers:
       username: ${OS_USERNAME}
       password: ${OS_PASSWORD}
       projectName: ${OS_PROJECT_NAME}
+      domainName: ${OS_PROJECT_DOMAIN_NAME}
```

Comment:
I have both properties OS_USER_DOMAIN_NAME and OS_PROJECT_DOMAIN_NAME defined locally (to the same value). Maybe you are correct and one of them is not needed. They are both valid however. https://docs.openstack.org/developer/python-openstackclient/authentication.html. Maybe add both of them? (With one of them commented out?)